### DISCUSSION TITLE
Help with equation-hovering issue

### DISCUSSION THREAD

**Discussion by yuvaltassa** [pradyunsg/furo#635] [Questions]
Please take a look at [this doc section](https://mujoco.readthedocs.io/en/latest/computation.html#numerical-integration) and hover over the equations. As you can see, the link-symbol-on-hover behavior is making the equation numbers jump (since the numbers are right-aligned and the link symbol appears to their right).

For reference, the source is [here](https://github.com/deepmind/mujoco/blob/81ba9ec54a8ed1dfd10386a6afb7b3555c244d56/doc/computation.rst?plain=1#L430).

Any idea how this can be fixed?

Legitimate options are:
1. Make the link symbol appear on the left, just for these links.
2. No link at all, we don't actually need the link, just the equation number, so we can refer to it in the text.

Any help would be appreciated.

**Comment by pradyunsg** [comment]
Oooh, this is an interesting one. Thanks for reporting.

I'm pretty sure it's due to `sphinxcontrib.katex`'s styles interacting weirdly with Furo's hover-management based styles.

**Reply by pradyunsg** [reply]
Yea, it's got styles to show the eqno and hoverlink (https://github.com/hagenw/sphinxcontrib-katex/blob/eebaa9517f4cd2041b752ff5a315f89664fea8bf/sphinxcontrib/katex-math.css#L40-L50) and Furo does as well (https://github.com/pradyunsg/furo/blob/48c0bf2aa983fb1ccd79b4167fe85387249afedf/src/furo/assets/styles/content/_math.sass) and they're conflicting.
